python-plugin/pom.xml

```diff
@@ -63,9 +61,25 @@
                     <pluginClass>fr.cnumr.python.CustomPythonRulesPlugin</pluginClass>
                     <sonarLintSupported>true</sonarLintSupported>
                     <sonarQubeMinVersion>${sonarqube.version}</sonarQubeMinVersion>
-                    <requirePlugins>python:${sonar.python.version}</requirePlugins>
+                    <!-- TODO : commented because plugin not loaded in SONAR if required version of python plugin is not installed -->
+                    <!-- TODO : check how to resolve it (how to force this version in SonarQube installation) -->
+                    <!-- <requirePlugins>python:${sonar.python.version}</requirePlugins> -->
+                    <basePlugin>python</basePlugin>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>${maven-shade-plugin.version}</version>
```

Comment:
please move this version management to root `pom.xml` file
1 - put these lines in root `pom.xml` file (in `pluginManagement` part like other maven plugins)
`
<plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-shade-plugin</artifactId>
                <version>${maven-shade-plugin.version}</version>
</plugin>
`
2 - delete in each langage module the line with de "version"

Reply:
done